### PR TITLE
Add reasoning for buffer alignment.

### PR DESCRIPTION
**Pull request by jdashg**
I wasn't sure on how to format this. I think this works.

### CODE REVIEW

File: specs/latest/1.0/index.html [hunk 1]
```diff
@@ -3362,6 +3362,18 @@ <h2 class="no-toc">Table of contents</h2>
 
 </p>
 
+<div class="note">
```
Comment (kenrussell) [inline specs/latest/1.0/index.html]: Olli just added the ability to specify the classes "note rationale" to get slightly different formatting. Could you use that here?

**Comment by kenrussell** [comment]
Could you also please update the last modified date at the top of the file?

For the record, this addresses Issue #914 which I prematurely closed and partially addresses Issue #916 .

**Comment by jdashg** [comment]
Fixed?

**Comment by kenrussell** [comment]
Looks good. Thanks.

**Pull request merged**